application/resources/views/admin/admin_users/edit.blade.php

```diff
@@ -0,0 +1,76 @@
+@extends('layouts.admin')
+
+@section('content')
+    <div class="row pt-3">
+        <div class="col-sm">
+            <form action="{{ route('admin.admin_users.update', $admin_user->id) }}" method="POST" enctype="multipart/form-data">
+                @csrf
+                @method('PUT')
+                <div class="form-group">
+                    <label for="name">名称</label>
+                    <input type="text" class="form-control @error('name') is-invalid @enderror" id="name" name="name"
+                           value="{{ old('name', $admin_user->name) }}" placeholder="名称" autofocus>
+                    @error('name')
+                        <span class="invalid-feedback" role="alert">
+                            <strong>{{ $message }}</strong>
+                        </span>
+                    @enderror
+                </div>
+
+                <div class="form-group">
+                    <label for="email">メールアドレス</label>
+                    <input type="text" class="form-control @error('email') is-invalid @enderror" id="email" name="email"
+                           value="{{ old('email', $admin_user->email) }}" placeholder="メールアドレス" autofocus>
+                    @error('email')
+                        <span class="invalid-feedback" role="alert">
+                            <strong>{{ $message }}</strong>
+                        </span>
+                    @enderror
+                </div>
+
+                <div class="form-group">
+                    <label for="password">パスワード</label>
+                    <input type="password" class="form-control @error('password') is-invalid @enderror" id="password" name="password" placeholder="パスワード" autocomplete="new-password">
+                    @error('password')
+                        <span class="invalid-feedback" role="alert">
+                            <strong>{{ $message }}</strong>
+                        </span>
+                    @enderror
+                </div>
+
+                <div class="form-group">
+                    <label for="password-confirm">パスワード(確認)</label>
+                    <input type="password" class="form-control @error('password-confirm') is-invalid @enderror" id="password-confirm" name="password_confirmation" placeholder="パスワード(確認)" autocomplete="new-password">
+                    @error('password-confirm')
+                        <span class="invalid-feedback" role="alert">
+                            <strong>{{ $message }}</strong>
+                        </span>
+                    @enderror
+                </div>
+
+                @if($admin_user->is_owner === false)
```

Comment:
```suggestion
                @if($admin_user->is_owner)
```

Reply:
で @else を上と入れ替えしたら？

Reply:
いや、こうすべきだな
```suggestion
                @can('changeAuthority', $admin_user)
```
当然、AdminUserPolictに該当するメソッドは足さないといけない